Formula/libtensorflow.rb

```diff
@@ -0,0 +1,46 @@
+class Libtensorflow < Formula
+  desc "C interface for Google's OS library for Machine Intelligence"
+  homepage "https://www.tensorflow.org/"
+  url "https://github.com/tensorflow/tensorflow.git", :tag => "v1.0.0",
+    :revision => "07bb8ea2379bd459832b23951fb20ec47f3fdbd4"
```

Comment:
I could not find a tgz. The "install from source" page pulls from github. @asimshankar, is there something I'm missing ?